modules/AWS/nfs/outputs.tf

```
@@ -1,3 +1,7 @@
 output "helm_release_nfs_service_ip" {
   value = data.kubernetes_service.nfs.spec[0].cluster_ip
+}
+
+output "nfs_pvc_claim_name" {
```

maybe simply `nfs_claim_name`

```suggestion
output "nfs_claim_name" {
```